README-dev.md

```diff
@@ -141,8 +141,23 @@ npm run test
 npm run test:unit
 ```
 
+#### Running Mina-Signer Tests
+`npm run build` is not recursive, thus in order for the Mina Signer tests to run you must execute the following from the root directory:
+
+```sh
+cd src/mina-signer
+npm run build
+cd ../..
+```
+
 This runs all the unit tests and provides you with a summary of the test results.
 
+Note that you can run individual jest tests via the command:
```

Comment:
optional comment here but I can't imagine outside devs don't ask this. could save some time. would have saved me some digging. up to you however.

Reply:
Useful thanks!